guides/get-started-microservices-on-kubernetes/simple-microservice-client-part1.adoc

```diff
@@ -0,0 +1,290 @@
+= {simple-microservice-client-part1}
+:summary: Invoke one microservice from another
+:includedir: ../_includes
+include::{includedir}/_attributes.adoc[]
+include::./_includes/_titles.adoc[]
+include::_includes/_constants.adoc[]
+// you can override any attributes eg to lengthen the
+// time to complete the guide
+:prerequisites-time: 10
+
+In this guide, you will learn HOW-TO invoke one microservice from another;
+
+[[prerequisites]]
+== Prerequisites
+
+To complete this guide, you need:
+
+* Complete link:simple-microservice-part1[{simple-microservice-part1}]
+
+== Introduction
+
+This guide is the first in a series of three:
+
+1. In link:simple-microservice-client-part1[{simple-microservice-client-part1}] (this guide), we explain HOW-TO invoke one microservice from another;
+2. In link:simple-microservice-client-part2[{simple-microservice-client-part2}], as usual, we explain HOW-TO run the whole thing on Kubernetes
+3. In link:simple-microservice-client-part3[{simple-microservice-client-part3}] we explain HOW-TO propagate user authentication and authorization data from the calling microservice to the invoked microservice; this is most useful in a scenario where you have a "chain" of microservices ("**A -> B -> C -> etc.**") and you want the user's authentication and authorization data to be propagated from one microservice to the next;
+
+== This Guide
+
+In these guides, we work with a simple invocation chain composed by +++<u>two</u>+++ microservices:
+
+* **Microservice A**: acting as **client**
+* **Microservice B**: acting as **server**
+
+Our invocation chain is then: "**Microservice A -> Microservice B**": when working with https://microprofile.io/[Microprofile], this is achieved by using the https://github.com/eclipse/microprofile-rest-client[microprofile-rest-client];
+
+Specifically, **Microservice A** will use the https://github.com/eclipse/microprofile-rest-client[microprofile-rest-client] to invoke the Jakarta REST service exposed by **Microservice B**;
+
+For both services, we will start from the microservice we built in link:simple-microservice-part1[{simple-microservice-part1}] (complete code in {source-code-git-repository}/simple-microservice);
+
+== Microservice B - the server
+
+We start from the server because we need the server's API for the client later on;
+
+=== Maven Project
+
+Copy {source-code-git-repository}/simple-microservice into a new folder named *simple-microservice-server* and:
+
+* remove folder *src/test*
+* remove all test scope dependencies
+
+NOTE: we remove tests in preparation for the upcoming guides
```

Comment:
@jamezp I changed the wording ... I was basically trying to say that the tests that were included with the archetype wouldn't be relevant anymore since we are using service to service invocation